### PR TITLE
Warning for deprecation of windows_zipfile resource

### PR DESCRIPTION
**Pull request by NAshwini**
Signed-off-by: NAshwini <ashwini.nehate@msystechnologies.com>

### Description

Warning for deprecation of windows_zipfile resource.

### Issues Resolved

https://github.com/chef-cookbooks/windows/issues/615

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Provides a set of Windows-specific resources to aid in the creation of cookbooks
 
 ### Deprecated Resources Note
 
-As of Chef 14.7+ the windows_share and windows_certificate resources are now included in the Chef Client. If you are running Chef 14.7+ the resources in Chef client will take precedence over the resources in this cookbook. In November 2019 we will release a new major version of this cookbook that removes these resources.
+As of Chef 14.7+ the windows_share and windows_certificate resources are now included in the Chef Client. Also the windows_zipfile resource is now included in the Chef 15.0.293+. If you are running Chef 14.7+ the resources in Chef client will take precedence over the resources in this cookbook. In November 2019 we will release a new major version of this cookbook that removes these resources.
```
Comment (tas50) [inline README.md]: windows_zipfile isn't included in Chef. It's been replaced by the new archive_file resource in Chef 15.0+

**Review by tas50 — CHANGES_REQUESTED** [review]
See the comment above

File: README.md (hunk 1 above)
Comment (NAshwini) [reply README.md]: Thank you @tas50, I missed it.

**Comment by NAshwini** [comment]
@tas50 Thank you I have fixed the review comment by commit https://github.com/chef-cookbooks/windows/pull/617/commits/70b8db92b54078914a0effb27830435a1f48c74d

File: README.md [hunk 2]
```diff
@@ -347,6 +347,8 @@ SeTakeOwnershipPrivilege             Take ownership of files or other objects
 
 ### windows_zipfile
 
+`Note`: This resource is now included in Chef 15.0.293 and later. There is no need to depend on the Windows cookbook for this resource.
```
Comment (tas50) [inline README.md]: This note will need to be updated as well to point users to archive_file

File: resources/zipfile.rb [hunk 3]
```diff
@@ -116,6 +116,7 @@
 
   def ensure_rubyzip_gem_installed
     require 'zip'
+    Chef::Log.warn('The windows_zipfile resource has been deprecated and will be removed from the next major release of the windows cookbook. Please update any cookbooks using this resource to instead `archive_file` resource if you are using Chef-15 and later.')
```
Comment (tas50) [inline resources/zipfile.rb]: ### 
```suggestion
    Chef::Log.warn('The windows_zipfile resource has been deprecated as Chef Infra Client 15.0 shipped with a new archive_file resource, which natively handles multiple archive formats. Please update any cookbooks using this resource to instead use the `archive_file` resource: https://docs.chef.io/resource_archive_file.html')
```

**Pull request merged**